pkg/api/upgrade/v20170930/orchestratorversiontypes.go

```diff
@@ -9,7 +9,6 @@ type OSType string
 //  - version: major, minor, and patch version numbers
 type OrchestratorProfile struct {
```

Comment:
`UpgradeProfile` and `PoolUpgradeProfile` needs to be moved out of API version 2017-09-30 and added to 2017-07-01 and 2017-08-31 versions because these procy child resource types under `containerservice` and `managedcluster` resource types.

Reply:
Can we do this later? I don't know the context of this. @dmitsh can you fix this after this PR merged?